front_end/html/summarize_logs.html

```diff
@@ -0,0 +1,158 @@
+
+<form method="post" action="/summarize_logs">
+
+<div class="columns">
+  <div class="column is-half">
+    <h3>CodeBuddy Summarized Logs</h3>
+    <div class="field is-grouped">
+      <div class="control">
+        <label class="label">Select File</label>
+        <div class="select is-info is-medium">
+          <select id="file_select" name="file_select">
+            <option selected disabled>Select File</option>
+            <option value="logs/summarized/HitsAnyUser.tsv.gz">HitsAnyUser</option>
+            <option value="logs/summarized/HitsPerUser.tsv.gz">HitsPerUser</option>
+            <option value="logs/summarized/LoadDuration.tsv.gz">LoadDuration</option>
+          </select>
+        </div>
+      </div>
+      <div class="control">
+        <label class="label">Filter Location</label>
+        <div class="select is-info is-medium">
+          <select id="filter_select" name="filter_select">
+            <option selected>No filter</option>
+            {% for item in filter_list %}
+              <option>{{ item }}</option>
+            {% end %}
+          </select>
+        </div>
+      </div>
+    </div>
+    <div class="control">
+      <label class="label">Filter by Date</label>
+      <div class="field is-grouped">
+        <div class="control">
+          <label class="label">Year</label>
+          <div class="select is-info is-medium">
+            <select id="year_select" name="year_select">
+              <option selected>No filter</option>
+              <option>20</option>
```

Comment:
Please come up with a way to populate the year dropdown dynamically rather than hard-coding the year. We want to avoid having to update this each year.